tool/ood-gen/lib/job.ml

```diff
@@ -0,0 +1,108 @@
+type metadata =
```

Comment:
Do you think we need some kind of date here like `published` to make it easier to sort by most recently added ?

Reply:
That's a good point. I don't think it's useful for sorting the job posts (we can just take the order as-is if people add their listing at the beginning of the file), but it'd be very useful in the UI (e.g. add a publication date, add a "new" tag, etc.)!